### PR TITLE
feat(cli): display the debug option more prominently in Terminal UI

### PR DESCRIPTION
**Pull request by EvanBacon**
# Why

Integrate better with SDK 47.

# How

- Ensure the debug option is always shown in the terminal UI for SDK +47.
- Update the docs to be a bit more concise.

# Test Plan

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by brentvatne — APPROVED** [review]
cc @Kudo

**Review by Kudo — APPROVED** [review]
looks good. thanks for making the instruction and doc clear.

i still have some changes to the using-hermes doc, e.g.
- remove the ios m1 workaround
- add the `No compatible apps connected.` a dedicated faq section to describe how it could happen.
- update `publishing updates` section, because the bundled hermes from react-native 0.69. the `hermes-engine` package is obsolete.

i'll create a separate pr based on this pr.

File: docs/pages/guides/using-hermes.mdx [hunk 1]
```diff
@@ -12,11 +12,18 @@ import { Terminal } from '~/ui/components/Snippet';
 
 A limitation with JavaScriptCore is that the debugger does not work with modules built on top of [JSI](https://github.com/react-native-community/discussions-and-proposals/issues/91). If your app uses [react-native-reanimated](https://github.com/software-mansion/react-native-reanimated) version 2, for example, [remote JS debugging will not work](https://docs.swmansion.com/react-native-reanimated/docs/#known-problems-and-limitations). Hermes makes it possible to debug your app even when using JSI modules.
 
-## Android setup
+## Support
 
-> Hermes for Android is supported from SDK 42 and above in Expo Go, [development builds](/development/introduction.mdx) using `expo-dev-client` and standalone apps built with EAS Build. For bare apps created before SDK 42, [follow these instructions to update your project configuration](https://expo.fyi/hermes-android-config).
+The Hermes Engine is fully supported across all Expo tooling, check if your project is using a compatible version:
```
Comment (amandeepmittal) [inline docs/pages/guides/using-hermes.mdx:17]:
```suggestion
The Hermes engine is fully supported across all Expo tooling. See the table below to check if your project is using a compatible version:
```

**Review by amandeepmittal — APPROVED** [review]
Thanks for updating this guide 👍 

Left one small suggestion.

**Pull request merged**